### PR TITLE
rails is fucking stupid

### PR DESCRIPTION
**Pull request by AskForTheWolf**
^^

### CODE REVIEW

**Comment by superbuggy** [comment]
can you remove the `.git` file in `todo_app`, add&commit again, then push up your changes? you created a rails app with a nested repo. when you pass `--git` to rails new, a repo is initialized within the rails app

**Comment by AskForTheWolf** [comment]
aggghhhh this keeps happening
Yeah i can

**Comment by AskForTheWolf** [comment]
To be honest, I'm not even sure how to fix it or stop this from keep
happening.  Like, git, why can't I just add some stupid files to a project?

**Comment by superbuggy** [comment]
Ah okay, the problem stems from a `.git` directory (this is what `git` uses to establish and track repositories) living inside of the `todo_app` directory, while `todo_app`'s parent folder is *also* being tracked as a repository; it too has a `.git` directory-- this was part of the `checkpoint-05` repo you cloned it down. `rm -rf .git` ***inside todo_app***. since this `.git` file was tracking your commit, you will then lose your commits. it will then be necessary to re-add and commit your files. after you've done this, push your commits.

**Comment by superbuggy** [comment]
Please re-submit by tomorrow @ 1:30

**Comment by AskForTheWolf** [comment]
Nah, it's giving me errors still

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.1.1'
+# Use postgresql as the database for Active Record
+gem 'pg', '~> 0.18'
```
Comment (superbuggy) [inline Gemfile:12]: 👍

File: db/migrate/20170606131710_create_todos.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateTodos < ActiveRecord::Migration[5.1]
+  def change
+    create_table :todos do |t|
+      t.string] :[content[
+      t.boolean]] :is_completed[
+
```
Comment (superbuggy) [inline db/migrate/20170606131710_create_todos.rb:6]:
```rb
create_table :todos do |t|
  t.string :content
  t.boolean :is_completed
  t.timestamps
end
```

**Pull request closed without merging**